R/trial_sequence.R

```diff
@@ -112,28 +113,15 @@ setMethod(
     catn("")
     catn("IPW for informative censoring:")
     show(object@censor_weights)
-  }
-)
-
-setMethod(
-  "show",
-  c(object = "trial_sequence_PP"),
-  function(object) {
-    callNextMethod()
-    catn("")
-    catn("IPW for treatment switch censoring:")
-    show(object@switch_weights)
-  }
-)
-
-setMethod(
-  "show",
-  c(object = "trial_sequence_AT"),
-  function(object) {
-    callNextMethod()
+    if (.hasSlot(object, "switch_weights")) {
+      catn("")
+      catn("IPW for treatment switch censoring:")
+      show(object@switch_weights)
+    }
+    # expansion
     catn("")
-    catn("IPW for treatment switch censoring:")
-    show(object@switch_weights)
+    show(object@outcome_model)
```

Comment:
How about a heading here?
```suggestion
    catn("Outcome model:")
    show(object@outcome_model)
```